src/components/UserIcon/UserIcon.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import classNames from "classnames";
+
+import "./_user-icon.scss";
+
+export default function User(props) {
+  const [userPanelVisibility, setUserPanelVisibility] = useState(false);
+
+  console.log(props);
+
+  const logOut = e => {
+    e.preventDefault();
+    console.log("logging out");
+    localStorage.removeItem("identity");
+    localStorage.removeItem("https://api.jujucharms.com/identity");
+    // TODO: Use React Router to redirect correctly
+    window.location.href = "/";
```

Comment:
I'm not sure this is necessary - the application should just render the 'login UI' once this has cleared. If that's not happening then we have a different problem.

If your intention was to navigate to the `/` just by matter of course I would add a `<link>` which would use the router to navigate to / but then attach an event handler to the click on that element which would first clear the items from local storage.

Reply:
The first point - it does not just render login.

Second point - that approach doesn't work when you're already on `/`